R/get_sections.R

```diff
@@ -67,81 +73,83 @@
 #'   }}
 #'   \itemize{
-#'   \item \code{environ}: a \code{dataframe} containing the environments present
-#'   within the section, with the following columns:
+#'   \item \code{environ}: a \code{dataframe} containing the environments
+#'     present within the section, with the following columns:
 #'   \itemize{
 #'      \item \code{name}: The named environment (e.g., "delta plain").
 #'      \item \code{type}: The named environment type (e.g., "siliciclastic").
 #'      \item \code{class}: The named environment class (e.g., "marine").
 #'      \item \code{prop}: The proportion of the environment within the section,
-#'      calculated from the individual Macrostrat units within the section.
-#'      \item \code{environ_id}: The unique identification number of the environment.
+#'        calculated from the individual Macrostrat units within the section.
+#'      \item \code{environ_id}: The unique identification number of the
+#'        environment.
 #'   }}
 #'   \itemize{
 #'   \item \code{econ}: a \code{dataframe} containing the economic attributes
-#'   present within the section, with the following columns:
+#'     present within the section, with the following columns:
 #'   \itemize{
 #'      \item \code{name}: The named economic attribute (e.g., "gold").
 #'      \item \code{type}: The named economic attribute type (e.g., "mineral").
 #'      \item \code{class}: The named economic attribute class (e.g., "precious
-#'      commodity").
-#'      \item \code{prop}: The proportion of the economic attribute out of potential
-#'      economic attributes contained within the section, calculated from the
-#'      individual Macrostrat units within the section.
+#'        commodity").
+#'      \item \code{prop}: The proportion of the economic attribute out of
+#'        potential economic attributes contained within the section, calculated
+#'        from the individual Macrostrat units within the section.
 #'      \item \code{econ_id}: The unique identification number of the economic
-#'      attribute.
+#'        attribute.
 #'   }
 #' }
 #'
-#' @author Christopher D. Dean
+#' @section Developer(s):
+#'   Christopher D. Dean
+#' @section Reviewer(s):
+#'   Bethany Allen
+#'
 #' @details More information can be found for the inputs for this function
 #' using the definition functions (beginning with \code{defs_}).
+#'
 #' @examples
 #' \dontrun{
-#'   # get_sections(section_id = 1)
-#'   # get_sections(column_id = 10)
-#'   # head(get_sections(age = 73))
-#'   # head(get_sections(age_top = 70, age_bottom = 75))
-#'   # head(get_sections(lat = 45, lng = -100))
-#'   # head(get_sections(lithology = "sandstone"))
-#'   # head(get_sections(environ_type = "siliciclastic"))
-#'   # head(get_sections(pbdb_collection_no = c(1000:5000)))
+#' # Get sections within a specified column
+#' ex1 <- get_sections(column_id = 10)
+#' # Get sections within a specified latitude-longitude box
```

Comment:
I think this is from a specific lat/long coordinate point, not an associated box.